client/extensions/woocommerce/app/store-stats/store-stats-referrer-widget/index.js

```diff
@@ -158,17 +162,32 @@ class StoreStatsReferrerWidget extends Component {
 							<TableRow
 								key={ d.referrer }
 								href={ href }
-								className={ classnames( { 'is-selected': selectedReferrer === d.referrer } ) }
+								className={ classnames( {
+									'is-selected': selectedReferrer === d.referrer,
+								} ) }
 							>
 								<TableItem isTitle>{ d.referrer }</TableItem>
 								<TableItem>
 									<HorizontalBar
-										extent={ extent }
+										extent={ extent_sales }
 										data={ d.sales }
 										currency={ d.currency }
 										height={ 20 }
 									/>
 								</TableItem>
+								<TableItem>
+									<HorizontalBar extent={ extent_views } data={ d.product_views } height={ 20 } />
+								</TableItem>
+								<TableItem>
+									<HorizontalBar extent={ extent_views } data={ d.add_to_carts } height={ 20 } />
+								</TableItem>
+								<TableItem>
+									<HorizontalBar
+										extent={ extent_views }
```

Comment:
I see that you are using the views extent so that the purchase events can be easily compared by size to the views block. But it begins to break down after a while. Over smaller data sets, this isn't as much of an issue, but i'm wondering about having the extent over the entire dataset as opposed to just the page in question. Does the ability to view from the perspective of the overall scale take too much away from having nice blocks?

![screen shot 2018-04-03 at 1 20 09 pm](https://user-images.githubusercontent.com/1922453/38224001-00cdcacc-3742-11e8-9d40-e90d0d31e861.png)

Reply:
The more I think about this the more I feel it was a mistake to include `views`, `carts` and `purchases` in this widget with all the confusing scales to interpret.

They're important stats but we can add another widget alongside or below the gross sales referrer widget and in another PR.